src/dvclive/huggingface.py

```diff
@@ -53,10 +71,11 @@ def on_train_end(
         control: TrainerControl,
         **kwargs,
     ):
-        if args.load_best_model_at_end:
-            trainer = Trainer(
+        if self._log_model == "last" and state.is_world_process_zero:
+            fake_trainer = Trainer(
                 args=args, model=kwargs.get("model"), tokenizer=kwargs.get("tokenizer")
             )
-            trainer.save_model()
-            self.live.log_artifact(args.output_dir)
+            output_dir = os.path.join(args.output_dir, "last")
+            fake_trainer.save_model(output_dir)
+            self.live.log_artifact(output_dir, type="model", copy=True)
```

Comment:
Cross-framework consistency isn't our highest priority, but should we agree on some common principles for the final artifact, like naming and whether to copy it?

Reply:
I think I would like for all the integrations to just have 2 options:

- `all/checkpoints`: resuming scenarios.
Log the entire checkpoint folder

- `best`: model registry
Log the best checkpoint on end with `copy=True, name="best", type="model")`

Reply:
That's fine with me. Do you want to update the lightning logger to use `copy=True`? AFAIK the rest is consistent.

Reply:
will update this and lightning to use that

Reply:
Sorry, are you also suggesting to change the behavior of `log_model=True` in lightning to track only the copied best artifact and not the whole directory? That's fine, just want to make sure I understand what you mean.

For HF, how should we handle the last/best checkpoint? If `args.load_best_model_at_end`, we could add `name=best`? WDYT?

Reply:
> or HF, how should we handle the last/best checkpoint? If args.load_best_model_at_end, we could add name=best? WDYT?

Updated with this behavior

Reply:
Also, dropped `last` option in favor of `True`